pagefaultgames/pokerogue
# [Balance] Update Egg Moves

## What are the changes?

New egg move changes to go along with the passive ability changes.

## Why am I doing these changes?



To have the egg move changes that goes along with the new passive abilities.

## What did change?


193 Pokémon got revised egg moves, Eternatus now has egg moves (still cannot be hatched yet).

## How to test the changes?




Simply look at the starter list to see all the new egg moves, start a run if necessary.

## Checklist
